spring-integration-xml/src/test/java/org/springframework/integration/xml/config/XPathRouterTests-context.xml

```diff
@@ -7,6 +7,15 @@
 		http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
 		http://www.springframework.org/schema/integration/xml http://www.springframework.org/schema/integration/xml/spring-integration-xml.xsd">
 
+	<int-xml:xpath-router id="parseOnly" input-channel="xpathRouterEmptyChannel"
+			order="2"
+			auto-startup="false"
+			role="foo"
+			timeout="123"
```

Comment:
I wonder if we should deprecated this and introduce a new `send-timeout` one.
To be consistent with all other `messagingTemplate-aware` components, like `splitter`, `filter` etc..